static/app/utils/replays/getCurrentUrl.tsx

```diff
@@ -20,11 +20,11 @@ function getCurrentUrl(
   const origin = initialUrl ? new URL(initialUrl).origin : '';
 
   const mostRecentNavigation = last(
-    navigationCrumbs.filter(({timestamp}) => +new Date(timestamp || 0) < currentTimeMs)
+    navigationCrumbs.filter(({timestamp}) => +new Date(timestamp || 0) <= currentTimeMs)
```

Comment:
is this going to be one of the first items in the array more often now?

Reply:
This fixed the issue in several cases. I still saw some missing a URL with the current default origin though.